<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntennaCalculator</class>
 <widget class="QWidget" name="AntennaCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>515</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AntennaCalculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="aircraftProfileLayout">
     <item>
      <widget class="QLabel" name="aircraftProfileLabel">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Aircraft Profile</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="aircraftProfileForm">
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="aobLabel">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>AOB</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="aobInput">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="suffix">
          <string comment=" ">°</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="altitudeLabel">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Altitude</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="altitudeInput">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="suffix">
          <string> ft</string>
         </property>
         <property name="maximum">
          <number>30000</number>
         </property>
         <property name="singleStep">
          <number>500</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="standOffLabel">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stand Off</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="standOffDisplay" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <widget class="QLabel" name="NmLabel">
          <property name="geometry">
           <rect>
            <x>74</x>
            <y>0</y>
            <width>21</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>NM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
         <widget class="QLCDNumber" name="standOffValue">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>75</width>
            <height>25</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="antennaConfigLayout">
     <item>
      <widget class="QLabel" name="antennaConfigLabel">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Antenna Config</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="antennaConfigForm">
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="downTiltLabel">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Down Tilt</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="downTiltInput">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="suffix">
          <string>°</string>
         </property>
         <property name="minimum">
          <number>15</number>
         </property>
         <property name="maximum">
          <number>75</number>
         </property>
         <property name="value">
          <number>45</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="beamWidthLabel">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Beam Width</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="beamWidthInput">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="suffix">
          <string>°</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="figLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
